kairon/shared/auth.py

```diff
@@ -79,8 +81,8 @@ async def get_current_user(
                 if iat_val is not None:
                     issued_at = datetime.utcfromtimestamp(iat_val)
                     if Utility.is_exist(
```

Comment:
timestamp__gte=issued_at, check_base_fields=False??
also move to AuditDataProcessor and call it is_password_reset()?